src/oatpp/parser/json/mapping/Deserializer.hpp

```diff
@@ -133,6 +133,7 @@ class Deserializer {
     if(caret.isAtText("null", true)){
       return oatpp::Void(T::Class::getType());
     } else {
+      //TODO: the below gives warning C4244: 'argument': conversion from 'v_int64' to 'TValueType', possible loss of data
```

Comment:
Example of data loss here:

```cpp
oatpp::String json = "256";
auto value = jsonObjectMapper->readFromString<oatpp::UInt8>(json); // UInt8 will be rounded to 0
```

The question is - do we care about such cases?

I think for now - we can just static cast the value to silent the error, and leave the corresponding `TODO info here`